apis/nodecore/v1alpha1/flavour_types.go

```diff
@@ -137,6 +137,9 @@ type FlavourSpec struct {
 	// This field is used to specify the optional fields that can be retrieved from the Flavour.
 	// In the future it will be expanded to include more optional fields defined in the REAR Protocol or custom ones.
 	OptionalFields OptionalFields `json:"optionalFields"`
+
+	// This field is used to represent the available quantity of transactions of the Flavour.
+	QuantityAvailable int `json:"quantityAvailable,omitempty"`
```

Comment:
What is the meaning of this field? Why do we need it?

Reply:
This paramenter is just a counter of the number of resources agregated. If the flavours are going to be an interface to the exterior it should be able to have information of how much resources are available.

Reply:
Ok, now I know what it is about but I would like to correct the comment as it goes as "available quantity of transactions" which is not clear what it means.

Nevertheless, I still don't understand who need this parameter, why and how.

Is it really well designed? What if a consumer want to buy a partition of this aggregation which is not a multiple of the original nodes of the provider's cluster?

I feel we need to talk about this better and deeper as it could be a braking change for the current and future architecture.

Reply:
Yes,  we need to have a deep conversation about how the flavour will behave when partitioned.